### PR TITLE
Travis scripts fix

### PR DESCRIPTION
**Pull request by miguelcleverti**
Travis fails when because of outdated scripts.
Demo content was removed so travis should no longer run try to run these.
Also changed EzPublishKernel into AppKernel in behat.yml configs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,8 @@ env:
   global:
     - CONFIG="behat.yml"
   matrix:
-      # test behat bundle with demo content
-    - CONFIG="vendor/ezsystems/behatbundle/EzSystems/BehatBundle/behat.yml" INSTALL="demoContentNonUniqueDB" PROFILE="default" TEST="behat"
-      # test demo with demo content
-    - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
+      # test behat default
+    - PROFILE="default" TEST="behat"
```
Comment (bdunogier) [inline .travis.yml]: While at it, I'd change this to `BEHAT_ARGS="--profile=default --suite=behat". It has been adopted on other repos, and I find it more straightforward.

**Comment by bdunogier** [comment]
The `behat.yml` file from this repository is NOT included from ezplatform's `behat.yml.dist`. It needs to be added there.

File: behat.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ default:
     extensions:
```
Comment (bdunogier) [inline behat.yml]: It doesn't make sense to have it there (none of the other included behat.yml files have it). This config is provided by the root behat.yml file.

You should remove the extension part altogether (check how it's done in platformui).

Comment (miguelcleverti) [inline behat.yml]: I agree!

**Comment by miguelcleverti** [comment]
@bdunogier changed the behat.yml to behat_suites.yml as discussed:
https://github.com/ezsystems/ezpublish-kernel/pull/1559
https://github.com/ezsystems/PlatformUIBundle/pull/495
https://github.com/ezsystems/ezplatform/pull/82

File: behat_suites.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# This file is for manual testing only, corresponding file in ezpublish-community is used for travis testing
```
Comment (andrerom) [inline behat_suites.yml]: Nitpick: ezpublish-community > ezplatform

**Comment by bdunogier** [comment]
Manually fixed the nitpick in master.

**Pull request merged**